components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -385,9 +389,12 @@ export const StyledWidgetToggle = styled<WidgetToggleProps, 'button'>('button')`
   float: ${p => p.float ? 'right' : 'none'};
   margin-right: ${p => p.float ? 10 : 0}px;
   border: none;
-  margin-top: 8px;
+  margin-top: 15px;
```

Comment:
nit for another time I suppose: `StyledWidgetToggle` seems very specific to the Cards section to just be in the general settings styles. Same for a few of these other components, e.g. `StyledSettingsInfo`.